### ISSUE TITLE
Prevent Social Buttons from Tracking by replacing them with placeholders

### ISSUE THREAD

**Issue by tildelowengrimm** [brave/brave-browser#534]
Social buttons on websites can track the user. We should replace them with inert placeholders which don't make outbound requests until a user interacts with them.

Replaces brave/browser-laptop#312

Edit bbondy: I'm landing this in 0.63.x which allows users to turn off social buttons and embedded tweets:
brave/brave-core#1818

**Comment by maciejwalkowiak** [comment]
If tracking via social buttons still works, it means that Facebook knows anyway most of the sites we visit - I am a bit surprised that this issue has such a low priority. What is the suggested way to block Facebook from tracking users in Braze browser until this issue is fixed?

**Comment by tomlowenthal** [comment]
@maciejwalkowiak You're right. We should just block them outright until we build the safe way to defang them. I've opened a new issue for the short-term fix: #3241.

**Comment by DaniellMesquita** [comment]
Any social widget should have a placeholder with its same size, and containing inside the site's favicon.
When clicking, it opens a browser modal which asks to enable feature if it is need, while telling about privacy.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
